examples/overriding-django-field-properties-with-InferExcept.ipynb

```
@@ -0,0 +1,192 @@
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "from typing import final\n",
+                "\n",
+                "from django.db import models\n",
+                "\n",
+                "\n",
+                "@final\n",
+                "class SomeModel(models.Model):\n",
+                "    some_field = models.CharField(max_length=100, help_text=\"Name\")\n",
+                "\n",
```

Ruff (0.11.9)

Use of `assert` detected

(S101)